source/settings/read-only.txt

```diff
@@ -0,0 +1,111 @@
+.. _compass-read-only:
+
+====================================
+Restrict Write Operations to MongoDB
+====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the ``readOnly`` option to prevent users from performing write 
+operations to MongoDB thorugh |compass-short|. If you set the ``readOnly`` 
+option, users cannot modify documents, create indexes, or specify validation 
+rules.
+
+About This Task 
+---------------
+
+By default, |compass-short| disables :guilabel:`Set Read-Only Mode`.
+
+If :guilabel:`Set Read-Only Mode` is enabled, |compass-short| automatically 
+disables the following options: 
+
+- :guilabel:`Enable MongoDB Shell`
+- :guilabel:`Enable DevTools`
```

Comment:
[consideration]

Rather than saying the settings that are disabled, could we maybe say what the result is? As in something like:

```
... |compass-short| automatically disables:

- MongoDB Shell in Compass
- Compass Developer Tools
```

Reply:
Also the compass dev tools page is in progress, so it'll be left unlinked for now, but I'll update with the link in the upcoming changes.